challenges/3-tipos-de-triangulo/javascript/ex3.js

```diff
@@ -0,0 +1,32 @@
+//Pai nosso que estais no céu faz esse código funcionar pfvr
+let A = process.argv[2]
+let B = process.argv[3]
+let C = process.argv[4]
+let lados = [A, B, C]
+lados.sort((a, b) => b - a);
+let ladoA = lados[0];
+let ladoB = lados[1];
+let ladoC = lados[2];
+console.log(ladoA, ladoB, ladoC);
+if (ladoA >= (Number(ladoB) + Number(ladoC))){
+    console.log("Não forma triângulo.")
+    return;
+  }
+  else {
+    if ((ladoA === ladoB) && (ladoB === ladoC)) {
+    console.log("triângulo equilátero.")
+  }
+  if ((ladoA === ladoB) || (ladoB === ladoC)) {
```

Comment:
Primo nesse condicional faltou verificar se um deles é diferente.
No estado atual desse condicional se você passar os 3 lados iguais ele vai considerar como `true`, o que não se aplica, pois um triângulo é isósceles apenas quando somente dois dos lados são iguais.

Reply:
Vlw, vou corrigir

Reply:
Oi eu fiquei com dúvida nisso e até adicionei ao final do código um comentário que falava sobre isso, mas se eu não me engano, todo triângulo equilátero também é isósceles, independentemente do terceiro lado ser igual. Até pesquisei pra ter certeza, mas se for necessário que diferencie o equilátero de isósceles então eu mudo.
![image](https://user-images.githubusercontent.com/74953869/194203065-12c32dcb-b9dc-4267-8f9a-ff46442cf2c3.png)